.eslintrc.js

```diff
@@ -38,6 +38,7 @@ module.exports = {
         'plugin:@typescript-eslint/recommended',
       ],
       rules: {
+        '@typescript-eslint/no-explicit-any': 'off',
```

Comment:
may be using the `warn` is more informative.
as this we will have info and we can avoid the `any` on some cases.

what do you think ?

Reply:
Yes we can let the warning